userspace/libsinsp/sinsp_public.h

```diff
@@ -19,7 +19,7 @@ limitations under the License.
 #pragma once
 
 #ifdef _WIN32
-#define SINSP_PUBLIC __declspec(dllexport)
+#define SINSP_PUBLIC
 #else
 #define SINSP_PUBLIC
 #endif
```

Comment:
How about we remove SINSP_PUBLIC completely then?

Reply:
that is an extremely valid point. SINSP_PUBLIC is a legacy from the pre-CMake times when sinsp was a dll in Windows. However, I will avoid making this change as part of a PR which is already huge and I will try to work on it separately. In the meantime, I removed the silly ifdef on the right.